index.js

```diff
@@ -34,10 +33,7 @@ const xappConfig = {
   secret: GRAVITY_SECRET,
 }
 
-xapp.init(xappConfig, () => {
-  config.GRAVITY_XAPP_TOKEN = xapp.token
-  bootApp() // eslint-disable-line
-})
+xapp.init(xappConfig, bootApp) // eslint-disable-line
```

Comment:
@alloy - I just ran into that auth issue that you mentioned a while back, and moved it down (scroll)